src/packagers/JSConcatPackager.js

```diff
@@ -72,17 +69,17 @@ class JSConcatPackager extends Packager {
 
       // If this was an ES6 export all (e.g. export * from 'foo'), resolve to the original exports.
       if (dep.isExportAll) {
-        asset.cacheData.exports[`$${asset.id}$exports`] = `$${mod.id}$exports`;
-        this.exports.set(`$${asset.id}$exports`, `$${mod.id}$exports`);
+        // asset.cacheData.exports[`$${asset.id}$exports`] = `$${mod.id}$exports`;
+        // this.exports.set(`$${asset.id}$exports`, `$${mod.id}$exports`);
```

Comment:
commented these for now. they were causing issues for me, and don't seem quite right anyway.